test/acceptance/features/steps/dboperator.py

```diff
@@ -14,8 +14,8 @@ class DbOperator():
 
     name = ""
     namespace = ""
-    operator_source_name = "db-operators"
-    operator_registry_namespace = "pmacik"
+    operator_source_name = "pmacik-olm-registry"
+    operator_registry_image = "quay.io/pmacik/olm:v1"
```

Comment:
Should we consider having the image at `quay.io/redhat-developer/sbo-olm:v1` ?

Reply:
> Should we consider having the image at `quay.io/redhat-developer/sbo-olm:v1` ?

There are 2 app registries actually in quay (the `redhat-developer` where sbo is pushed, and `pmacik` where sample db operators are pushed) at this moment.

So `quay.io/redhat-developer/sbo-olm:v1` would be better for SBO itself and so the DB operators registry image would need a different name since it comes from different app registry.

So, what about creating the two image registries, one for SBO and other for the sample DB operators (`quay.io/redhat-developer/sbo-olm:v1` and `quay.io/redhat-developer/sample-db-operators-olm:v1`) and push the db operators registry image to the `sample-db-operators-olm` instead?

Reply:
yeah +1 for sample-db-operators-olm